src/main/kotlin/com/bridgecrew/initialization/InitializationService.kt

```diff
@@ -25,6 +25,7 @@ private val LOG = logger<InitializationService>()
 class InitializationService(private val project: Project) {
 
     private var isCheckovInstalledGlobally: Boolean = false
+    private var checkovVersion: String = "2.3.186"
```

Comment:
maybe it should be under `CheckovSettingsState` (without the UI, or at least without the ability to edit it), because it's a general attribute that affects the entire app and can't be changed by the user.

Reply:
@mayshavit WDYT?